jmix-flowui/flowui/src/main/resources/io/jmix/flowui/view/layout.xsd

```diff
@@ -4221,6 +4228,25 @@
         </xs:complexContent>
     </xs:complexType>
 
+    <xs:complexType name="rangeInputComponent">
```

Comment:
Take a look at `component_xml_load.HtmlComponentXmlLoadTest` and let's add test for RangeInput too